apps/web/src/app/[locale]/challenge/_components/comments/comment.tsx

```diff
@@ -121,8 +121,9 @@ export function Comment({
     hasNextPage: hasMoreReplies,
     refetch,
   } = useInfiniteQuery({
+    initialPageParam: 0,
```

Comment:
[initialPageParam](https://tanstack.com/query/v5/docs/react/guides/migrating-to-v5#infinite-queries-now-need-a-initialpageparam)